build.gradle

```diff
@@ -117,6 +117,7 @@ allprojects {
 
   repositories {
     jcenter()
+    mavenLocal()
```

Comment:
is this for resolving locally built snapshot versions?

Reply:
yeah - removed now